pkg/gridtypes/zos/gw_fqdn.go

```diff
@@ -17,14 +17,20 @@ var (
 
 type Backend string
 
-// check if valid http://x.x.x.x:port or [::]:port
+// check if valid http://ip:port, http://ip or ip:port
```

Comment:
I would pass the `passthrough` flag to the Valid method (may be also make it private method) hence the workload validation will also validate that this backend matches the tls-passthrough requirements. hence give an early validation error when validating the workload before it attempts to deploly its.

Note that the workload validation is done ealry and also done in the client side. hence clients can catch the issue before it's sent to the node